.github/workflows/messaging.yml

```diff
@@ -148,16 +148,15 @@ jobs:
       matrix:
         target: [ios, tvos, macos --skip-tests]
         flags: [
-          '--use-modular-headers',
-          '--skip-tests --use-libraries'
+          '--use-modular-headers'
         ]
     needs: pod-lib-lint
     steps:
     - uses: actions/checkout@v2
     - name: Setup Bundler
       run: scripts/setup_bundler.sh
     - name: PodLibLint Messaging Cron
-      run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseMessaging.podspec --platforms=${{ matrix.target }}
+      run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseMessaging.podspec --platforms=${{ matrix.target }} --allow-warnings
```

Comment:
Why is ` --allow-warnings` added? Are the warnings fixable?

Reply:
Yep, so the messaging tests have an [intentional warning](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseMessaging/Tests/UnitTestsSwift/FIRMessagingAPITest.swift#L44-L52) that [fails a linting](https://github.com/firebase/firebase-ios-sdk/runs/5591176589?check_suite_focus=true).

Reply:
Hmmm, Maybe we should run two pod lib lints so we can still catch warnings in the library?

One with `--skip-tests` and one with `--allow-warnings`

Reply:
Discussed offline. Added `@unkown default` to silence warning. Resolved in latest work.